zebra-state/src/service/finalized_state/disk_format/upgrade/no_migration.rs

```diff
@@ -0,0 +1,48 @@
+//! An implementation of [`DiskFormatUpgrade`] for marking the database as upgraded to a new format version.
+
+use crossbeam_channel::Receiver;
+
+use semver::Version;
+use zebra_chain::block::Height;
+
+use crate::service::finalized_state::ZebraDb;
+
+use super::{CancelFormatChange, DiskFormatUpgrade};
+
+/// Implements [`DiskFormatUpgrade`] for pruning duplicate Sapling and Orchard note commitment trees from database
```

Comment:
Wrong docs?

Reply:
Yep, thank you, fixed in https://github.com/ZcashFoundation/zebra/pull/9263/commits/55b36028eb45e0349f56f7ebb24088f1f6c53e4e.